forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -55,12 +59,12 @@ internal class BTPinCollector(
                     )
                 )
             } catch (e: JSONException) { }
+            logger.i("Received successful response from BasisTheory")
             return ForageApiResponse.Success(forageResponse.toString())
         }
 
-        // This should return the BT ApiException, which is a string value of the API response
-        // TODO: Log this value to Datadog.
-//        val btErrorResponse = response.exceptionOrNull()
+        val btErrorResponse = response.exceptionOrNull()
+        logger.e("Received BasisTheory API exception on balance check: $btErrorResponse")
```

Comment:
include `merchantAccount`,  `paymentMethodRef` and `cardToken` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274193522

should omit cardToken